### ISSUE TITLE
11 februari 2025 - Overleg onderzoeksbureaus

### ISSUE THREAD

**Issue by Caitboard** [WCAG-Audit-Discussions/notulen-obnl#9]
11 februari 2025

## Aanwezig
- Jeroen Hulscher - Logius
- Caitlin de Rooij - Level Level
- Savitri Sinnema - Frameless
- Brian Bors - St. Accessibility
- Armen Mnatsian - Digitaal EHBO
- Els Groffen - Gemeente Nijmegen
- Erik Kroes - Erik Kroes
- Gerard Copinga - Cardan
- Iacobien Riezebosch - Logius
- Janita Top - Janita Top
- Koen Willems - Logius
- Leon Bouterse - Accessible Minds
- Niek Derksen - wcag.nl
- Robbert - NLDS / Frameless
- Roel Antonisse - Digitaal Toegankelijk
- Sander Nijsingh - Alverdae
- Eva Hess - Swink
- Jules Ernst - Logius

## Model toegankelijkheidsverklaring
Koen Willems verstrekt informatie over het model toegankelijkheidsverklaring, waar momenteel aan gewerk wordt:

Het [Besluit Digitale Toegankelijkheid Overheid](https://wetten.overheid.nl/BWBR0040936/2018-07-01#Hoofdstuk2_Artikel4), hierna "BDTO", kent 2 verplichtingen
1. Websites en apps toegankelijker maken.
2. Status van toegankelijkheid vastleggen in een verklaring (vergelijkbaar met etiket voedingswaarden).

Punt 2 moet overeenkomstig zijn met het door de minister vastgestelde model. Dat model bestaat momenteel (nog) niet. Op dit moment wordt op het ministerie van BZK gewerkt aan de vaststelling van de zogeheten modeltoegankelijkheidsverklaring. In de bijlagen van de mail van Jeroen op 10 februari vind je de concepten.

2 zaken springen eruit:
1. Onevenredige last
2. Uitgezonderde content

### Onevenredige last
In het BDTO staat dat als er een beroep wordt gedaan op een onevenredige last, er een toegankelijk alternatief aangeboden moet worden. Momenteel kan dat in de invulassistent vrij makkelijk omzeild worden. Voortaan moet dit ook duidelijk benoemd worden in de toegankelijkheidsverklaring. Zodat een bezoeker met beperking dit ook duidelijk kan vinden in die verklaring.

### Uitgezonderde content
Voorbeelden van uitgezonderde content:
- PDFs die gepubliceerd zijn voor 23 september 2018
- Online kaarten en karteringsdiensten

[Lees hier alle uitzonderingen](https://www.digitoegankelijk.nl/wetgeving/uitzonderingen)

Er moet in een toegankelijkheidsverklaring beschreven worden welke content er uitgezonderd is, omdat een gebruiker dit zelf niet weet wanneer ze op een pagina komen. Dit zal worden gedaan door concreet te vragen waar de uitgezonderde content uit bestaat.

### Procedure

Dit is een ministerieel besluit: Ministerie van Binnenlandse Zaken en Koninkrijksrelaties (BZK). 
- De staatssecretaris stelt dit vast. Hiervoor wordt nog een [internet consultatie](https://www.internetconsultatie.nl/) gehouden. Iedereen kan dit online zien en hierop een reactie achterlaten. Dit is momenteel nog niet gestart. 
- De duur van zo'n proces is ongeveer 6 weken. 
- Daarna gaat de besluitvormingspijplijn in. 
- Daarop volgt een memo naar het kabinetsberaad. 
- Daarna plaatst de staatssecretaris een handtekening. 
- Verwachting: 1 oktober dit jaar van kracht. 

Dit heeft geen invloed op de onderzoeksrapporten, wel op de toegankelijkheidsverklaringen.

Jeroen geeft een seintje wanneer de internetconsultatie geopend wordt. 

### Is er gekeken naar het optimaliseren van de vastgelegde informatie, zodat het waarde biedt voor toegankelijkheid? / Zijn er best practices van hoe dit ingevuld moet worden in de verklaring?

Koen geeft aan dat er ruimte nodig is voor men om hierin een eigen modus te vinden en wil dit niet dicht timmeren. De focus ligt op het wettelijke minimum, want dat is de wettelijke taak.

Robbert geeft aan dat het fijn zou zijn als er een standaard plek gecreëerd wordt binnen de toegankelijkheidsverklaring, zodat een gebruiker dit gemakkelijker terug kan vinden. 

Koen en Jeroen beamen dat dit de verantwoordelijkheid is van de webmaster en referen terug aan het "wettelijke minimum".

Nederland is een decentrale eenheidsstaat. Het rijk mag een lagere overheid pas iets opleggen als dat wettelijk beslist is. Overheidsinstanties moeten zich aan deze wet houden. Ministers mogen pas ingrijpen als dat in de wet zo beschreven is. Daarom blijft de focus op het wettelijke minimum. Hieraan moeten we ons minimaal houden. Overige zaken kunnen wij dus niet opleggen aan anderen. 

## Discussie: Logo's als user interface components afkeuren onder SC 1.4.11
Kan een logo afgekeurd worden op 1.4.11 als het een user interface component is met te weinig kleurcontrast? 

[WCAG 1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) zegt het volgende:
- Graphical objects kunnen als uitzondering gezien worden, wanneer het een logo betreft (which is considered "essential" to its presentation)
- User Interface components worden niet als uitzondering gezien

Er wordt specifiek gesproken over logo's met onvoldoende kleurcontrast, die als link naar de homepage dienen, in het hoofdmenu.

Brian geeft aan dat hij het niet zou afkeuren als:
- Er een alternatief is, dus een andere link naar de homepage.
- Delen van het logo wel duidelijk contrast hebben, waardoor de functie nog helder is.

Er wordt gezocht naar consensus over het wel/niet afkeuren hiervan. Op de W3C GitHub is er op een soortgelijke discussie geen consensus bereikt. [Logo excepted from 1.4.3 Contrast (Minimum), but does it fail as a link for 1.4.11 Non-text Contrast?](https://github.com/w3c/wcag/issues/1739)

### Tegen- en voorargumenten over het afkeuren
Tegen argumenten:
- Er wordt gesteld dat het primaire doel is dat het een logo is: de herkenning van het "merk" van de website. Dat valt onder een uitzondering en hoeft niet aan de contrast eis te voldoen.
- Er zal te veel weerstand ontstaan, wanneer we verplichten dat bij logo's rekening gehouden moet worden met kleurcontrasten. "Zo krijg je de wereld niet mee".
- Er is geen consensus op de W3C GitHub. We willen als Nederland niet te ver afwijken. 
- Er zijn alternatieven om weer op de homepage te komen, bijvoorbeeld door de URL aan te passen in de browser.

Voor argumenten:
- Dat het logo een link is, is misschien niet het primaire doel, maar het is wel degelijk een link en daardoor een User Interface Component, waar WCAG aan refereert.
- Er zijn alternatieven om op de homepage te komen, maar die vind je in de browser. Niet op de website zelf. Moet er dan niet op de website zelf een toegankelijk alternatief zijn?

### Consensus
Er is consensus bereikt dat dit *niet* afgekeurd wordt, onder WCAG 1.4.11. De nadruk ligt op de verwoording: "we keuren het niet af", in plaats van "we keuren het goed". 

- [ ] Actiepunt Brian: Bij W3C linken naar deze discussie
- [ ] Actiepunt Jeroen: Benoemen bij discussiegroep
